doc/sphinx/user/run-aspect/parameters-overview/structure.md

```diff
@@ -3,7 +3,7 @@
 Most of the run-time behavior of ASPECT is
 driven by a parameter file that looks in essence like this:
 
-``` prmfile
+```{literalinclude} ../../../../manual/cookbooks/overview/doc/structure.part.prm
```

Comment:
```suggestion
```{literalinclude} ../../manual/cookbooks/overview/doc/structure.part.prm
```

Reply:
I don't think this cookbook is in the cookbooks directory, only in the manual/cookbooks/overview/doc